common/build/build-common/ts-common-config.json

```diff
@@ -15,6 +15,6 @@
 		"types": [],
 		"strict": true,
 		"pretty": true,
-		"lib": ["DOM", "DOM.Iterable"]
+		"lib": ["ES2020", "DOM", "DOM.Iterable"]
```

Comment:
Please hold off on merging this until we can chat tomorrow. Thanks!

Reply:
I re-read my notes this morning and realized that @CraigMacomber and I agreed with @jatgarg earlier this month that we should make breaking changes to build-common in `next`. Approving and merging.